lib/core/text/flat.nit

```diff
@@ -582,6 +580,9 @@ class FlatBuffer
 
 	private var capacity = 0
 
+	# Real items, used as cache for when to_cstring is called
+	private var real_items: nullable NativeString = null
```

Comment:
a `nullable NativeString` is always boxed. You can remove the box by: 1. using a `noautoinit` NativeString and 2. an additional boolean flag to check the presence/validity of your cache